go.mod

```diff
@@ -4,7 +4,7 @@ go 1.12
 
 replace github.com/prometheus/client_golang => github.com/prometheus/client_golang v0.9.4
 
-replace golang.org/x/crypto => github.com/munnerz/crypto v0.0.0-20191203200931-e1844778daa5
+replace golang.org/x/crypto => github.com/meyskens/crypto v0.0.0-20200821143559-6ca9aec645f0
```

Comment:
Sorry to be a pain, but it makes me nervous to replace this stdlib crypto library with a fork.
But if we must do it, I think:
 * the fork should live in the cert-manager organization on Github
 * the reason for the fork should be explained briefly here, and in more detail in the fork's README.
 * make sure that the fork does not fall behind the upstream repo.
 * all changes should have links to  corresponding upstream issues and pull requests so that upstream can give feedback on our proposed changes.
 * Ensure that we do thorough code review and testing of our changes in the fork.

Reply:
> the fork should live in the cert-manager organization on Github

Agreed, this should have been done when we first forked this, i don't think this fork was meant to stay for long but unfortunately we have had no feedback yet on our EAB PR.

> the reason for the fork should be explained briefly here, and in more detail in the fork's README.

Disagree, this causes a commit that makes it harder to sync between upstream and commit back to upstream, the fork isn't meant to stay once merged into upstream. We should add a note in the GitHub description however

> make sure that the fork does not fall behind the upstream repo.

Sure thing

> all changes should have links to corresponding upstream issues and pull requests so that upstream can give feedback on our proposed changes.

EAB already has this done, this change will be PRed upstream as soon as it is reviewed by our team.

> Ensure that we do thorough code review and testing of our changes in the fork.

See reviewers note ;)